frontend/src/components/common/CompareDemographicsCommon.vue

```diff
@@ -398,6 +411,12 @@ export default {
         mergedFromPen = selectedStudents[0];
       }
 
+      // Same Pen validation
+      if (this.validateSamePen(mergedToPen, mergedFromPen,
+        'Error! PENs cannot be merged, as same PENs are selected.')) {
+        return;
```

Comment:
the validateSamePen accepts objects are these pen or student objects? if objects could we rename them

Reply:
rename is done.